### ISSUE TITLE
[BUG] dask-cudf 0.7.0 nightly package fails import

### ISSUE THREAD

**Issue by randerzander** [rapidsai/dask-cudf#226]
Fresh conda env, created as:
```
conda env create --name rapids --file /conda/environments/rapids_dev.yml
```
rapids_dev.yml:
```
name: rapids_dev
channels:
- rapidsai-nightly/label/cuda10.0
- nvidia
- conda-forge
- numba
dependencies:
- python>=3.6,<3.8
- cudatoolkit=10.0
- distributed>=1.23.0
- cmake
# rapids packages
- nvstrings<=0.8.0
- cudf=<0.8.0
- dask-cuda=<0.8.0
- dask-cudf=<0.8.0
```

Importing cudf is fine.

Importing dask_cudf fails to find libcudf_cffi.
```
Python 3.7.3 | packaged by conda-forge | (default, Mar 27 2019, 23:01:00) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import cudf
>>> import dask_cudf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/conda/envs/rapids/lib/python3.7/site-packages/dask_cudf/__init__.py", line 1, in <module>
    from .core import (
  File "/conda/envs/rapids/lib/python3.7/site-packages/dask_cudf/core.py", line 19, in <module>
    from libgdf_cffi import libgdf
ModuleNotFoundError: No module named 'libgdf_cffi'
```

**Comment by mrocklin** [comment]
cc @kkraus14 @raydouglass

**Comment by kkraus14** [comment]
@mrocklin working on this, will have a PR into dask-cudf to resolve shortly.

**Comment by randerzander** [comment]
closed by #228
